editor/src/core/shared/math-utils.ts

```diff
@@ -136,6 +136,12 @@ export function forceFiniteRectangle<C extends CoordinateMarker>(
   throw new Error('invariant: we expected a finite Rectangle')
 }
 
+export function canvasRectangleOrZeroRect(
```

Comment:
haha I think you are looking for `zeroRectIfNullOrInfinity`

Reply:
lol, that's the one!

Reply:
https://github.com/concrete-utopia/utopia/pull/4519/commits/eb3b899eb2c238cf0e7b057e2fb3b7aaf2bf1d18